stackstate_checks_base/tests/test_http_helper.py

```diff
@@ -0,0 +1,458 @@
+import unittest
+import json
+from schematics.models import Model, DataError
+from schematics.types import StringType, IntType, BooleanType
+from stackstate_checks.utils.http_helper import HTTPHelper, HTTPRequestType, HTTPAuthenticationType, HTTPResponseType
+from stackstate_checks.utils.http_helper import HTTPHelperRequestHandler, HTTPHelperSessionHandler, HTTPMethod
+from stackstate_checks.utils.http_helper import HTTPHelperResponseHandler, HTTPHelperConnectionHandler, \
+    HTTPHelperRequestModel, HTTPHelperConnectionModel, HTTPHelperSessionModel, HTTPHelperResponseModel
+
+"""
+Structures used within the test cases
+"""
+
+
+class BodySchematicTest(Model):
+    hello = IntType(required=True)
+
+
+class BodyRequestSchematicTest(Model):
+    hello = StringType(required=True)
+    pong = BooleanType(required=True)
+
+
+class BodyResponseSchematicTest(Model):
+    hello = StringType(required=True)
+    pong = BooleanType(required=True)
+
+
+"""
+ HTTP Helper Request Class
+"""
+
+
+class TestHTTPHelperRequestHandler(unittest.TestCase):
+    def test_get_request_model(self):
+        request_handler = HTTPHelperRequestHandler()
+        model = request_handler.get_request_model()
+        assert isinstance(model, HTTPHelperRequestModel)
+
+        # Test the base model. This should fail as a few things are required
+        try:
+            request_handler.validate_request_model()
+        except DataError as e:
+            assert len(e.errors) == 2
```

Comment:
You are actually meant to use these assert methods. https://docs.python.org/3/library/unittest.html#assert-methods 

https://stackoverflow.com/a/45947566/1860591

Reply:
I followed the structure for most of the other integration libs, will replace